eth_tester/main.py

```diff
@@ -605,7 +605,7 @@ def create_log_filter(self, from_block=None, to_block=None, address=None, topics
 
         if is_integer(raw_from_block):
             if is_integer(raw_to_block):
-                upper_bound = raw_to_block
+                upper_bound = raw_to_block + 1
```

Comment:
I thought I had fixed this in https://github.com/ethereum/eth-tester/pull/121.
How did this re-appear?

Reply:
does this PR need a rebase?

Reply:
Hello!

I `reset` this module to the wrong branch when this PR was a WIP.  I rebased a fix.
Good catch, and Thanks for the review. :cowboy_hat_face: